src/client/src/orangehrmRecruitmentPlugin/components/CandidateAutocomplete.vue

```diff
@@ -0,0 +1,84 @@
+<!--
```

Comment:
Any reason for adding specific candidate autocomplete? can't we re-use employee autocomplete ?

Reply:
yes we can